_non-cosmos/polygon/assetlist.json

```diff
@@ -277,6 +277,49 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
         }
       ]
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "type_asset": "erc20",
+      "address": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+      "denom_units": [
+        {
+          "denom": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "page",
+          "exponent": 8
+        }
+      ],
+      "base": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+      "name": "Page",
+      "display": "page",
+      "symbol": "PAGE",
+      "traces": [
+        {
+          "type": "additional-mintage",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e"
+          }
```

Comment:
The PR check failed because the 'provider' is not specified here but is a required property for all non-ibc trace objects. I think something like `"provider": "PageDAO"` would be appropriate.